src/Aquifer.API/Endpoints/Projects/ProjectResourceStatusCounts.cs

```diff
@@ -4,44 +4,35 @@ namespace Aquifer.API.Endpoints.Projects;
 
 public class ProjectResourceStatusCounts
 {
-    private readonly List<ResourceContentStatus> _inManagerReviewStatuses =
+    internal static readonly List<ResourceContentStatus> InManagerReviewStatuses =
```

Comment:
Nit: `HashSet<T>` is more performant (`O(1)`) than `List<T>` (`O(N)`) when doing a `Contains()` call.  In these cases there are only two items in each list so it won't have a significant impact but this _is_ a performance related PR. 😄

Reply:
Probably not since it's getting translated to a SQL query.

Reply:
Ah, good point.  I'm not used to thinking in terms of `IQueryable`.